cmake/onnxruntime_training.cmake

```diff
@@ -13,8 +13,20 @@ file(GLOB_RECURSE onnxruntime_training_srcs
     "${ORTTRAINING_SOURCE_DIR}/core/framework/communication/*"
     "${ORTTRAINING_SOURCE_DIR}/core/session/*.h"
     "${ORTTRAINING_SOURCE_DIR}/core/session/*.cc"
+    "${ORTTRAINING_SOURCE_DIR}/core/agent/*.h"
+    "${ORTTRAINING_SOURCE_DIR}/core/agent/*.cc"
 )
 
+# TODO: Merge onnxruntime_training_srcs and onnxruntime_training_agent_srcs
+#       when onnxruntime_MINIMAL_BUILD removes the full training code
+if (onnxruntime_MINIMAL_BUILD)
```

Comment:
This is not needed either. 
Let's just don't worry about minimal build in this PR. 

we will discuss with Inference team on what's the proper setting when merge this into master.